bin/wasm-node/javascript/src/index.js

```diff
@@ -230,9 +230,17 @@ export function start(config) {
       return chainAddedPromise;
     },
     terminate: () => {
-      worker.terminate();
-      if (!workerError)
-        workerError = new AlreadyDestroyedError();
+      clearTimeout(livenessTimeout)
```

Comment:
```suggestion
      if (livenessTimeout !== null)
        clearTimeout(livenessTimeout)
```

I know if shouldn't change anything, but I'm not a fan of calling functions with bad parameters because the function says it's ok.

Reply:
The [spec says it's ok, though](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-cleartimeout)... But I don't mind changing that.